### PR TITLE
navigator: set acceptance radius even for IDLE waypoint

### PR DESCRIPTION
**Pull request by RomanBapst**
This fixes the heading drift people have been seeing right after a takeoff.
The problem was that the auto flighttask was creating a yaw setpoint in the direction of the velocity setpoint. There is code implemented which locks the yaw setpoint if the vehicle is within the acceptance radius of the target, however, the navigator published an IDLE setpoint with an acceptance radius of 0 and this is what was used by the flight task.

Still not sure why the acceptance radius was not updated once the takeoff setpoint was sent.

### CODE REVIEW

**Comment by RomanBapst** [comment]
@MaEtUgR FYI

**Comment by RomanBapst** [comment]
Before this PR:
image

After this PR:
yaw_good

**Comment by dagar** [comment]
This seems kind of weird and out of place. What if instead we did used `Navigator::reset_triplets()` in that section of logic to set appropriate defaults in all the position setpoints, and then set it to `position_setpoint_s::SETPOINT_TYPE_IDLE`?

**Comment by MaEtUgR** [comment]
@RomanBapst Please mention the original issue when you make a pr for a fix. It was this one: https://github.com/PX4/Firmware/issues/13675

Your findings explain why the yaw setpoint is not exactly the yaw estimate to begin with. But it doesn't explain why the yaw setpoint is quickly unfiltered jumping to zero and otherwise gets heavily filtered even though it's a reset. At least the jump to zero is a concern to me.

Is there an easy way to reproduce the issue in SITL such that it happens every takeoff? That would help debugging the other issues.

**Comment by RomanBapst** [comment]
@MaEtUgR Have a look at the plot above with the fix. I don't see any yaw clamping anymore.
I investigated why the yaw setpoint was moving around in arbitrary directions in the first place. That was obviously causes by the flighttask doing it's velocity vector heading calculations which was in turn caused by the acceptance radius being publishes as 0 by the navigator.

If you still see any clamping issues let me know, I do not see them.
Any by the way, the problem was easily reproducible in SITL. The only reason why you did not see it 100% of the time was because sometimes by coincidence the yaw setpoint was actually close to the current yaw during takeoff and so the effect was not really visible.

**Comment by RomanBapst** [comment]
@dagar I followed your suggestion, please have a look.

File: src/modules/navigator/navigator_main.cpp [hunk 1]
```diff
@@ -837,6 +834,10 @@ Navigator::reset_triplets()
 	_pos_sp_triplet.previous.valid = false;
 	_pos_sp_triplet.next.valid = false;
 	_pos_sp_triplet_updated = true;
+	_pos_sp_triplet.previous.acceptance_radius = get_default_acceptance_radius();
+	_pos_sp_triplet.current.acceptance_radius = get_default_acceptance_radius();
+	_pos_sp_triplet.next.acceptance_radius = get_default_acceptance_radius();
+
```
Comment (MaEtUgR) [inline src/modules/navigator/navigator_main.cpp]:
```suggestion
```

**Pull request merged**